### ISSUE TITLE
Let's Encrypt

### ISSUE THREAD

**Issue by tnorthcutt** [Fyrd/caniuse#2041]
It would be rad to add info for https://letsencrypt.org/ support

**Comment by rahul286** [comment]
:+1: :-)

**Comment by defries** [comment]
👍

**Comment by dannyvankooten** [comment]
:thumbsup:

**Comment by Ugoku** [comment]
There's nothing special about their certificates, so what is the reason you want to have a separate entity for Let's Encrypt?

I just tried [their example page](https://helloworld.letsencrypt.org) in IE8 on Windows XP (in a VM though) and it is accepted there. Unless the VM has an influence on the acceptance of certificates, this leads me to believe their certificates are well supported everywhere.

**Comment by rahul286** [comment]
@Ugok http://browsershots.org/https://helloworld.letsencrypt.org/ shows that it is not working in older browser.

Test case is simple. http://browsershots.org/https://helloworld.letsencrypt.org/ should open without SSL warning.

**Comment by Ugoku** [comment]
Hm, that's weird. Firefox 25-30 seem to fail but 24 and even Firefox 4 work fine. Could be something else that's unrelated? I'm not convinced yet.

**Comment by Fyrd** [comment]
I agree with @Ugoku, and even looking at http://browsershots.org/https://helloworld.letsencrypt.org/ it seems virtually all browsers seem to support it without warning? Are there any particular browser versions that are known to not support the Let's Encrypt certificates that can be verified?

**Comment by okj579** [comment]
The list of root CAs depends mostly on your operating system. As of [now](https://letsencrypt.org/2015/10/19/lets-encrypt-is-trusted.html), Let's Encrypt certificates should be trusted in all major OSs and browsers, even old ones.

This is really a Let's Encrypt issue, not a browser support one.

**Comment by rahul286** [comment]
When I shared  http://browsershots.org/https://helloworld.letsencrypt.org/ some older firefox browsers were failing. I think there might be some update from browsershots or Let's Encrypt.

**Comment by Fyrd** [comment]
Okay, closing this issue.

**Comment by SimonEast** [comment]
There is a good list of supported/unsupported browsers here, which I think would be a worthy addition to caniuse.com:

https://community.letsencrypt.org/t/which-browsers-and-operating-systems-support-lets-encrypt/4394

...especially considering the massively long thread which is increasingly hard to follow as time goes on.

@Ugoku said "There's nothing special about their certificates", however that's not strictly true. The browser (or in some cases the operating system) needs to support SHA256 certificates *plus* have the *DST Root CA X3* root certificate installed, which is not the case in many older browsers.

Some documents mention requiring support for SNI, although I'm not fully sure if SNI is *required* to use a LetsEncrypt certificate, or only when deploying it on a shared IP address (I suspect the latter?). If it's the latter, then SNI support is a completely separate issue.
